autoload/composer.vim

```diff
@@ -39,7 +39,8 @@ endfunction
 function! s:json_decode(string) abort
```

Comment:
Let's also go ahead and rename the function's `string` parameter to `expr` per Neovim's docs for `json_decode()`.